### PR TITLE
fixes #18048 - fix ostree and docker cv pages

### PR DESCRIPTION
**Pull request by thomasmckay**
The ostree content view page was not rendering an "add" repo button. The controller for both ostree and docker were also outdated and not triggering calls to the server to fetch already-added repos.

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-ostree-repositories.html [hunk 1]
```diff
@@ -25,90 +23,67 @@ <h3 translate>
...
+            ng-show="isState('content-view.repositories.ostree.available') && permitted('edit_content_views', contentView)"
+            ng-click="addRepositories(contentView)">
+      <span translate>Add Repositories</span>
+    </button>
   </div>
 
   <span data-block="no-rows-message">
     <span ng-show="isState('content-view.repositories.ostree.list')" translate>
-      There are currently no OSTree Repositories associated with this Content View, you can add some by clicking on the "Add" tab above.
+      There are currently no Repositories associated with this Content View, you can add some by clicking on the "Add" tab above.
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-ostree-repositories.html]: Why remove the word OSTree?

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-docker-repositories-list.controller.js [hunk 2]
```diff
@@ -22,16 +22,24 @@ angular.module('Bastion.content-views').controller('ContentViewDockerRepositorie
             'organization_id': CurrentOrganization,
             'content_view_id': $scope.$stateParams.contentViewId,
             'content_type': 'docker'
-
         },
         'queryUnpaged');
 
-        nutupane.masterOnly = true;
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-docker-repositories-list.controller.js]: You'll definitely want this in there or you will get errors about not defining `closeTable()`.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-ostree-repositories-list.controller.js [hunk 3]
```diff
@@ -22,16 +22,24 @@ angular.module('Bastion.content-views').controller('ContentViewOstreeRepositorie
             'organization_id': CurrentOrganization,
             'content_view_id': $scope.$stateParams.contentViewId,
             'content_type': 'ostree'
-
         },
         'queryUnpaged');
 
-        nutupane.masterOnly = true;
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-ostree-repositories-list.controller.js]: See as above comment.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-ostree-repositories.html [hunk 4]
```diff
@@ -1,12 +1,10 @@
 <span page-title ng-model="contentView">{{ 'Repositories for Content View:' | translate }} {{ contentView.name }}</span>
 
 <header data-block="sub-header">
-  <h3 translate>
-    OSTree Repository Selection
-  </h3>
+  <h3 translate>OSTree Repositories</h3>
 </header>
 
-<nav data-block="navigation">
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-ostree-repositories.html]: Why did you remove this data block reference here?

Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-ostree-repositories.html]: The changes to ostree.html come directly from file.html: If it is not in ostree it was not in file. Any fixes I would want to make in both places. Please advise.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-ostree-repositories.html (hunk 1 above)
Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-ostree-repositories.html]: The changes to ostree.html come directly from file.html: If it is not in ostree it was not in file. Any fixes I would want to make in both places. Please advise.

Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-ostree-repositories.html]: That doesn't really answer the question.  Aren't OSTree Repositories distinct from yum repositories?  To me having this extra descriptor was useful.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-ostree-repositories.html (hunk 4 above)
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-ostree-repositories.html]: They should both be using data-block="navigation"

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-docker-repositories-list.controller.js (hunk 2 above)
Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-docker-repositories-list.controller.js]: as discussed on IRC, I did not add this back

**Comment by thomasmckay** [comment]
@waldenraines - updated

**Comment by thomasmckay** [comment]
[test]

**Comment by thomasmckay** [comment]
[test]

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-docker-repositories-list.controller.js (hunk 2 above)
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-docker-repositories-list.controller.js]: I forget what we decided on IRC but I still think it's possible this will generate a JS error about close not being defined.

Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-docker-repositories-list.controller.js]: added masterOnly back to both controllers
